cloud_functions/gcs_event_based_ingest/Dockerfile.ci

```diff
@@ -0,0 +1,6 @@
+FROM python:3.8-slim
+COPY requirements.txt requirements-dev.txt .
+RUN pip3 install -r requirements-dev.txt
+WORKDIR /ci
+COPY . /ci/
+ENTRYPOINT ["pytest"]
```

Comment:
Trailing newline

Reply:
fixed also a good cue that this dockerfile should also be linted :) I've also added a hadolint step to cloudbuild.